tests/components/onewire/test_entity_owserver.py

```diff
@@ -418,6 +430,8 @@ async def test_owserver_setup_valid_device(hass, device_id):
     # Ensure enough read side effect
     read_side_effect.extend([ProtocolError("Missing injected value")] * 10)
 
+    # Ignore default_disabled for testing
+    DEVICE_SENSORS["26"][2]["default_disabled"] = False
```

Comment:
It's safer to patch objects when we need to change them during tests. We may want to have the original behavior in a future test that we add at some point and then this may cause a hard to find bug.

Reply:
@MartinHjelmare, that was the plan, and it worked fine on my machine (python 3.8) 
```
    patch.dict(
        "homeassistant.components.onewire.sensor.DEVICE_SENSORS"
    ) as device_specs
            device_specs["26"][2]["default_disabled"] = False
```

but then it failed in CI for test 3.7: https://github.com/home-assistant/core/runs/1349250998

I can try to fix it as part of https://github.com/home-assistant/core/pull/42832 if you have some ideas of why it wasn't working.